tests/integration/test_fim/test_files/test_recursion_level/test_recursion_level.py

```diff
@@ -21,15 +21,17 @@
 dir_no_recursion = os.path.join(PREFIX, 'test_no_recursion')
 dir_recursion_1 = os.path.join(PREFIX, 'test_recursion_1')
 dir_recursion_5 = os.path.join(PREFIX, 'test_recursion_5')
-dir_recursion_320 = os.path.join(PREFIX, 'test_recursion_320')
+dir_recursion_320 = os.path.join(PREFIX, 'test_recursion_32') if sys.platform == "win32" else os.path.join(PREFIX, 'test_recursion_320')
 subdir = "dir"
 
 dir_no_recursion_space = os.path.join(PREFIX, 'test no recursion')
 dir_recursion_1_space = os.path.join(PREFIX, 'test recursion 1')
 dir_recursion_5_space = os.path.join(PREFIX, 'test recursion 5')
-dir_recursion_320_space = os.path.join(PREFIX, 'test recursion 320')
+dir_recursion_320_space = os.path.join(PREFIX, 'test recursion 32') if sys.platform == "win32" else os.path.join(PREFIX, 'test recursion 320')
 subdir_space = "dir "
 
+recursion_320 = 32 if sys.platform == "win32" else 320
```

Comment:
Change the name of the variable to a generic name because this can be confusing.